### PR TITLE
disable search preview

### LINKED ISSUE #1888: How to disable the search interface
Hello docs team
We still have questions on the support side about how to disable the search interface, although it's written on the documentation
See, https://github.com/meilisearch/meilisearch/discussions/2778

I think it's indeed not really obvious to find the answer to this question on the documentation
Maybe it could be worth putting it on the [getting started](https://docs.meilisearch.com/learn/getting_started/quick_start.html#search-preview) or in the [FAQ](https://docs.meilisearch.com/faq.html)?

Again just suggestions 😄

### PR DESCRIPTION
**Pull request by maryamsulemani97**
closes #1888

### CODE REVIEW

File: learn/getting_started/quick_start.md [hunk 1]
```diff
@@ -269,6 +269,10 @@ Meilisearch offers a search preview where you can preview search results. It com
 
 ![Meilisearch's search preview showing the movies index](/search_preview/default.png)
 
+::: note
+For security reasons, the search preview is only available in [development mode.](/learn/configuration/instance_options)
```
Comment (maryamsulemani97) [inline learn/getting_started/quick_start.md:273]:
```suggestion
For security reasons, the search preview is only available in [development mode.](/learn/configuration/instance_options.md#environment)
```

File: learn/getting_started/quick_start.md [hunk 2]
```diff
@@ -269,6 +269,10 @@ Meilisearch offers a search preview where you can preview search results. It com
 
 ![Meilisearch's search preview showing the movies index](/search_preview/default.png)
 
+::: note
+For security reasons, the search preview is only available in [development mode.](/learn/configuration/instance_options.md#environment)
+:::
```
Comment (dichotommy) [inline learn/getting_started/quick_start.md:274]: These custom containers (note, tip, warning etc.) are mainly useful for making text stand out from their surroundings. In this case there's no surrounding text, so I personally don't think it's necessary.

Other than that, it's a nice addition! LGTM.

```suggestion
For security reasons, the search preview is only available in [development mode.](/learn/configuration/instance_options.md#environment)
```

**Review by dichotommy — APPROVED** [review]
LGTM! 🍀

**Comment by maryamsulemani97** [comment]
bors merge

**Pull request merged**